Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -268,6 +274,9 @@ function WebMapServiceImageryProvider(options) {
   pickFeatureResource.setQueryParameters(pickFeatureParams, true);
 
   this._resource = resource;
+
+  // TODO: change this pickFeatureResource to something like options.getFeaturesUrl if this option exists, else use the getCapabilities default URL
+  // UPDATE: a new paramter getFeatureInfoUrl has been introduced, if the parameter is found in options, the pickFeaturesRource will refer to the paramter
```

Comment:
I don't think this UPDATE comment is needed.

Reply:
Thank you @ebogo1 , I will remove the comment as well.